src/server.cc

```diff
@@ -673,8 +672,8 @@ InferenceServer::PrintBackendAndModelSummary()
       }
     }
   }
-  std::string models_table_string = models_table.PrintTable();
-  LOG_INFO << models_table_string;
+
+  LOG_TABLE_INFO(models_table);
```

Comment:
`LOG_INFO_NOESCAPE(models_table.PrintTable())`? Seems to be not clear if calling out the object type twice..

Reply:
resolving based on discussion - will revisit after